src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -143,16 +143,20 @@ public static Set<Tag> parseTags(Collection<String> tags) throws ParseException
      * Leading and trailing whitespaces will be trimmed.
      */
     public static Price parsePrice(String price) {
-        BigDecimal decimalPrice = new BigDecimal(price);
-        return new Price(price, decimalPrice);
+        requireNonNull(price);
+        String priceTrimmed = price.trim();
+        BigDecimal decimalPrice = new BigDecimal(priceTrimmed);
+        return new Price(priceTrimmed, decimalPrice);
     }
 
     /**
      * Parses {@code String area} into a {@code Area}.
      * Leading and trailing whitespaces will be trimmed.
      */
     public static Area parseArea(String area) {
-        int intArea = Integer.parseInt(area);
+        requireNonNull(area);
+        String areaTrimmed = area.trim();
+        int intArea = Integer.parseInt(areaTrimmed);
```

Comment:
Just wondering how we will be handling the NumberFormatException that Integer.parseInt() throws? Is this error being handled?

Reply:
+1. You may need to add a MESSAGE_NOT_AREA?

Reply:
@lrongyi only bug to fix and you can merge sweetie cheeks